pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/options/impl/ParallelsAlwaysFailFast.java

```
@@ -45,7 +45,7 @@ public ParallelAlwaysFailFast() {
         @Override
         @Nonnull
         public String getDisplayName() {
-            return "parallelsAlwaysFailFast Set the quiet period for the job";
+            return "Parallels stage Always FailFast";
```

I'd go with something like `Parallel stages always fail fast`